docs/products/kafka/kafka-connect/howto/manage-connector-versions.md

```diff
@@ -69,6 +72,16 @@ as part of an Aiven for Apache Kafka service, use the [Aiven API](https://api.ai
 [Aiven Provider for Terraform](https://registry.terraform.io/providers/aiven/aiven/latest/docs)
 to set the connector version.
 
+:::warning
```

Comment:
```suggestion
:::note
```
Not sure this needs to be a warning. I think a note fits better here. We usually reserve warnings for major breaking changes.